docs/_docs/troubleshooting.md

```diff
@@ -206,6 +206,14 @@ you don't have a proper JavaScript runtime. To solve this, either install
 `execjs` and `therubyracer` gems, or install `nodejs`. Check out
 [issue #2327](https://github.com/jekyll/jekyll/issues/2327) for more info.
 
+### Timeout Error
+
+If you get a timeout error after `bundle install`, check [Is It Up](http://isitup.org/rubygems.org) to see that the RubyGems website is up.
+
+If it is, open the Gemfile in your site's folder and change the first line from `https://www.rubygems.org` to `http://www.rubygems.org`
```

Comment:
I don't think recommending folks switch away from `https` to `http` is a good thing to do here. Would you be willing to remove this line?

Reply:
@mattr- No problem, I understand that it was probably more of a network than Jekyll issue! I experienced the timeout error consistently and had to do a bit of Googling to see how to complete the installation.